util/dftool_module.py

```diff
@@ -20,7 +20,7 @@
     query,
 )
 from util.predictoor import csvs as predictoor_csvs
-from util.predictoor.query import queryPredictoors
+from util.predictoor.queries import queryPredictoors
```

Comment:
Why? It's almost always better to use singular not plural.

And this is definitely one of those cases imo

Reply:
I didn't see the other one I guess.

It should be singular everywhere. query.py

Reply:
Not sure why my previous comment does not appear, it seems to be pending and I can't publish it. Weird.
Here is an example: https://softwareengineering.stackexchange.com/a/75929. Based on this answer, I would argue that a single e.g. "volume" module contains multiple queries, multiple csvs (csv functions), multiple models. Therefore the module is singular because it handles volume-related things, but the "queries" file contains various queries, etc. by analogy models and csvs

Reply:
Good point, we do have plural in some other places.

So ok then we can call it queries.

Rather than have huge changes in one big PR, can we do this with more incremental PRs? Keep this one as is if you like, but in the future it will be more tractable.

Reply:
I'll handle the change after @trizin merges this PR.